modules/apps/asset/asset-categories-admin-web/src/main/java/com/liferay/asset/categories/admin/web/internal/display/context/AssetCategoriesDisplayContext.java

```diff
@@ -683,6 +684,13 @@ public List<DropdownItem> getVocabulariesDropdownItems() {
 		return DropdownItemListBuilder.add(
 			dropdownItem -> {
 				dropdownItem.putData("action", "deleteVocabularies");
+				dropdownItem.putData(
```

Comment:
@DiegoHu97, don't forget to add the redirect URL. It should be the entry point to categories admin (like when we click Categories in the product menu)